kiosk/src/Components/AddingGame.tsx

```diff
@@ -127,10 +132,9 @@ const AddingGame: React.FC<IProps> = ({ kiosk }) => {
                 <div className="addInstructions">
                     <h2>How to upload your game</h2>
                     <ol>
-                        <li>Scan the QR code to the right with your phone or webcam</li>
-                        <li>Open the link to scan your game's QR code</li>
-                        <li>Return to the Kiosk Main Menu</li>
-                        <li>You will find your game to the left of the GalgaMulti game</li>
+                        <li className="firstStep">Scan the QR code to the right with your phone<br/>-OR-<br/>Click the link under the QR code</li>
```

Comment:
I would just skip the following text. If the user doesn't have phone. They will go the link. We don't need to be explicit. 
OR-<br/>Click the link under the QR code</li